apps/com.google.AndroidStudio/eos-android-studio.desktop.in

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Version=1.0
+Name=Get Android Studio
+Exec=/usr/bin/eos-install-app-helper --app-id com.google.AndroidStudio --remote flathub --required-archs x86_64 %U
```

Comment:
```suggestion
Exec=/usr/bin/eos-install-app-helper --app-id com.google.AndroidStudio --remote flathub --branch stable --required-archs x86_64 %U
```

See https://github.com/endlessm/eos-application-tools/pull/42 where I discovered this is needed.